llvm/docs/CMake.rst

```diff
@@ -654,6 +654,11 @@ enabled sub-projects. Nearly all of these variable names begin with
   Generate dSYM files and strip executables and libraries (Darwin Only).
   Defaults to OFF.
 
+**LLVM_ENABLE_EXPORTED_SYMBOLS**:BOOL
+  When building executables, preserve symbol exports. Defaults to ON. 
+  You can use this option to disable exported symbols on all executable build
```

Comment:
```suggestion
  Setting this option to ``OFF`` removes exported symbols from all executables.
```

Reply:
Nit: Documentation tends not to use "You" as the subject.

Reply:
That's what I initially thought as well but a lot of the LLVM_INCLUDE* variables on the lines below follow a "You can use this option..." And they also have similar behavior which defaults to ON.

Reply:
Well if there's precedent I guess you can go either way. I'm personally not a fan but you should go with whatever you prefer.